tutorials/sac-story-explore-1-open/sac-story-explore-1-open.md

```diff
@@ -0,0 +1,128 @@
+
+Last feature to note here are the Search options available for finding content.
+
+![Search Bar](Search.png)
+
+  1. The **search** icon located on the Shell Bar is used for content across the whole SAP Analytics Cloud product. The Shell Bar is always visible at the top, making it easy for you to search while in a story.
+  2. The **search bar** below the Shell Bar allows you to find content within any of the folders or subfolders of **My Files**. For example, use this and search for "Sample – Revenue Analysis" to find the sample story.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Open and Explore the Sample Story)]
+Now that you understand the files and folders system, open the **"Sample - Revenue Analysis"** Sample Story within the **Samples** folder or search for it using the Search bar.
+
+<!-- After clicking into the story, you'll find yourself looking at this: -->
+
+<!-- ![Sample](Sample.png)
+![Sample](Sample resized.png) -->
+
+Let's quickly revisit what stories are and why they're important. Stories are much like an analytics dashboard or report that allow stakeholders to analyze, report, and plan with the historical data of a business. Furthermore, it provides instant access and augmented analysis for monitoring and measuring business performance and metrics.
+
+<!-- ![Analysis](Analysis.png) -->
+![Analysis resized](Analysis resized.png)
+
+This Story shows the "data story" of a beverage business called "The Best Run Juice Company" and demonstrates the performance of the business with different visualizations and features:
+
+| Number          |  Visualization / Feature         | Description
+|  :------------- |  :-------------                  | :-------------
+|1                |  **Numeric Point and Bullet Charts** | Visualizes the status of Key Performance Indicators (KPI).  You can immediately see the current KPI value compared to your business's performance thresholds.
+|2                |  **Time Series Chart**               | Displays trends in any kind of numeric data over time.  Is your business trending positively?  
+|3                | **Input Control Filters**            | Allows you to focus in on data segments.  Slide the Date Range filter to select your dates for analysis and the filter checkboxes helps you be precise in your analysis.
+|4                |  **Bar Charts**                      | Visualizes values across a Series of data.  As an example, the first chart on the left shows Gross Margin % by the Product and Location dimensions. They're easy to understand and interact with by drilling down, sorting, or analyzing with Smart Insights.  We'll use these together in the later tutorials.
+
+<!--
+1. Numeric Point Charts
+2. Bullet Chart
+3. Input Control Filters
+4. Time Series Chart
+5. Bar Charts -->
+
+[DONE]
+[ACCORDION-END]
+
+<!--
```

Comment:
Please remove tutorials parts that are not used.

Reply:
Done. Thanks